### PR TITLE
chore: add missing translation keys on service page

### PR DESCRIPTION
**Pull request by milljoniaer**
There have been some english translation keys missing on the service page:

Before:
image

After:
image

### CODE REVIEW

**Review by hofmann-felix — APPROVED** [review]
Thank you for taking care of it 🙏

I'll give you an approval, but wait with the merge until someone else has accepted it.

File: src/assets/locales/en/translations.json [hunk 1]
```diff
@@ -123,9 +123,10 @@
   "services.platform.list.infrastructure": "Infrastructure & Automation",
   "services.consulting.title": "Consulting",
   "services.consulting.teaser": "The experience of our employees is our most valuable treasure.",
-  "services.consulting.text": "The experience of our employees is our most valuable treasure. We share it with you and advise you in technical and conceptual matters. conceptual matters.",
```
Comment (georgiee) [inline src/assets/locales/en/translations.json:126]: oh a classic deepl error I think. it tends to repeat things at the end of sentences

File: src/assets/locales/en/translations.json [hunk 2]
```diff
@@ -123,9 +123,10 @@
...
+  "services.consulting.text": "The experience of our employees is our most valuable treasure. We share it with you and advise you in technical and conceptual matters.",
   "services.consulting.list.workshops": "Training & Workshops",
-  "services.consulting.list.analysis": "Technical & UX/UI Analysis",
+  "services.consulting.list.techanalysis": "Technical Analysis",
+  "services.consulting.list.uxanalysis": "UX/UI Analysis",
```
Comment (georgiee) [inline src/assets/locales/en/translations.json:129]: thanks

**Review by georgiee — APPROVED** [review]
Thanks for the summary and the include images.  Made it super easy to follow.

**Pull request merged**